Bottom: f7b6651f7950a2356525cce7d8d238bcb4ecb209
Top:    c649c7fb44e510d4d0b26a902a2cea337070b445
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-03 23:33:22 +0100

Refresh of refer-to-git-sources-not-cvs-w

---

diff --git a/src/parse_y.y b/src/parse_y.y
index bbb7255..b76e689 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -1916,7 +1916,7 @@ check_file_version (int ver)
     Message ("ERROR:  The file you are attempting to load is in a format\n"
 	     "which is too new for this version of pcb.  To load this file\n"
 	     "you need a version of pcb which is >= %d.  If you are\n"
-	     "using a version built from cvs sources, the source date\n"
+	     "using a version built from git source, the source date\n"
 	     "must be >= %d.  This copy of pcb can only read files\n"
 	     "up to file version %d.\n", ver, ver, PCB_FILE_VERSION);
     return 1;
